autogpt_platform/frontend/src/components/tokens/border-radius.stories.tsx

```diff
@@ -14,54 +14,54 @@ const meta: Meta = {
 export default meta;
 
 // Border radius scale data based on Figma design tokens
-// Custom naming convention: xs, s, m, l, xl, 2xl, full
+// Custom naming convention: xsmall, small, medium, large, xlarge, 2xlarge, full
 const borderRadiusScale = [
   {
-    name: "xs",
+    name: "xsmall",
     value: "0.25rem",
     rem: "0.25rem",
     px: "4px",
-    class: "rounded-xs",
+    class: "rounded-xsmall",
     description: "Extra small - for subtle rounding",
   },
   {
-    name: "s",
+    name: "small",
     value: "0.5rem",
     rem: "0.5rem",
     px: "8px",
-    class: "rounded-s",
+    class: "rounded-small",
     description: "Small - for cards and containers",
   },
   {
-    name: "m",
```

Comment:
Initially, these were named like so on Figma, I didn't realise they conflict with Tailwind native tokens.... For example, `rounded-s` is `border-start-start-radius` 😅 . 

These new names should create less conflict. Save to update as this was only used on design system components, which until now were not used on the app... 💆🏽